msal/tests/Test.MSAL.NET.Unit/CacheTests/TokenCacheTests.cs

```diff
@@ -252,20 +244,18 @@ public void GetAccessTokenExpiryInRangeTest()
                 ClientId = TestConstants.ClientId
             };
 
-            MsalAccessTokenCacheItem atItem = new MsalAccessTokenCacheItem()
-            {
-                Authority = TestConstants.AuthorityHomeTenant,
-                Environment = TestConstants.ProductionEnvironment,
-                ClientId = TestConstants.ClientId,
-                Scopes = TestConstants.Scope.AsSingleString(),
-                TokenType = "Bearer",
-                RawClientInfo = MockHelpers.CreateClientInfo(),
-                ExpiresOnUnixTimestamp = CoreHelpers.DateTimeToUnixTimestamp(DateTime.UtcNow + TimeSpan.FromMinutes(4))
-            };
-            atItem.CreateDerivedProperties();
+            var atItem = new MsalAccessTokenCacheItem(
+                TestConstants.ProductionEnvironment,
+                TestConstants.ClientId,
+                "Bearer",
+                TestConstants.Scope.AsSingleString(),
+                TestConstants.Utid,
+                "",
+                new DateTimeOffset(DateTime.UtcNow + TimeSpan.FromMinutes(4)),
+                MockHelpers.CreateClientInfo());
```

Comment:
Seems like these are almost duplicated. What's the reason not to put this in a method or set of methods, that potentially re-uses the same base and just sets a different dateimeoffset on the at?
e.g.:
GetExpiredAT()
{ return GetAT(expiredTime) }
GetAT(DateTimeOffset()...